api/CSPViolationReportBody.json

```diff
@@ -590,7 +590,7 @@
       "toJSON": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSPViolationReportBody/toJSON",
-          "spec_url": "https://www.w3.org/TR/CSP3/#cspviolationreportbody",
+          "spec_url": "https://w3c.github.io/webappsec-csp/#cspviolationreportbody",
```

Comment:
This `toJSON` entry should be removed, but I'll take care of that as a follow up.

Reply:
I've filed https://github.com/w3c/webappsec-csp/issues/546, it should be added to the spec.